src/detect.h

```diff
@@ -1227,6 +1227,9 @@ typedef struct DetectEngineThreadCtx_ {
     AppLayerDecoderEvents *decoder_events;
     uint16_t events;
 
+    /** stats id for lua rule errors */
+    uint16_t lua_rule_errors;
```

Comment:
Is u16 enough ?

Reply:
this is the count id

side thought: we should probably use a dedicated type for it

Reply:
> this is the count id

Thanks Victor :-)